src/dotty/tools/dotc/transform/LazyVals.scala

```diff
@@ -82,12 +78,11 @@ class LazyVals extends MiniPhaseTransform with SymTransformer {
       * with a LazyHolder from
       * dotty.runtime(eg dotty.runtime.LazyInt)
       */
-    def transformLocalValDef(x: ValDef)(implicit ctx: Context) = x match {
-      case ValDef(name, tpt, _) =>
+    def transformLocalValDef(x: DefDef)(implicit ctx: Context) = {
```

Comment:
The naming is a bit confusing since it is a `DefDef` that's transformed. Replace with `transformLocalGetter`?